### ISSUE TITLE
Pre submit build status in github is not refreshed though the build has finished

### ISSUE THREAD

**Issue by keyonghan** [flutter/flutter#83743]
`Linux fuchsia_precache` status was stuck for more than 2 hours for https://github.com/flutter/flutter/pull/83718 and commit: https://github.com/flutter/flutter/pull/83718/commits/bb267529516eb278e6489bbf38a5e9e0df3a933d.
However the build has finished: https://ci.chromium.org/ui/p/flutter/builders/try/Linux%20fuchsia_precache/16509/overview

From gcloud, there is no luci status refresh log about the above build.

To summarize, the build was scheduled/run in buildbucket, but status was not reflected in github UI.

/cc @yjbanov

**Comment by godofredoc** [comment]
obsolete
